tools/proteore_filter_keywords_values/filter_kw_val.xml

```diff
@@ -0,0 +1,271 @@
+<tool id="MQoutputfilter" name="Filter by keywords and/or numerical value" version="2019.06.27">
```

Comment:
Maybe `name="Filter tabular"` and `by keywords and/or numerical value` as description?